src/ResourceManager/Network/Commands.Network/ChangeLog.md

```diff
@@ -19,6 +19,8 @@
 --->
 
 ## Current Release
+
+## Version 6.6.0
```

Comment:
Can we move up the comment on ```* Changed default models representation to table-view``` into this release, and make sure it is in AzureRM.psd1 and AzureRM.Network.psd1?

Reply:
Done